### PR TITLE
Password check

### PR DESCRIPTION
**Pull request by mICH-C4C**

### CODE REVIEW

File: homework/password-check/validation.cpp [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+
+bool doPasswordsMatch(const std::string& pwd1, const std::string& pwd2) {
+    if (pwd1.compare(pwd2) == 0) {
+        return true;
+    }
+    return false;
+}
+ErrorCode checkPasswordRules(const std::string& pwd ) {
+    auto resultRules = ErrorCode::Ok;
+    const std::string patternOfLetters = "AĄBCĆDEĘFGHIJKLŁMNŃOÓPRSŚTUWYZŹŻ";
```
Comment (BarTes8) [inline homework/password-check/validation.cpp:31]: I think password need to have only latin letters

**Comment by ziobron** [comment]
Thanks @BarTes8 for Code Review.
🏅 1 XP granted

**Comment by ziobron** [comment]
Thanks @BarTes8 for Code Review.
🏅 1 XP granted

**Comment by ziobron** [comment]
We are closing this PR. If you want to fix your code and get points, please re-open it or raise a new PR

**Pull request closed without merging**